ntbs-build/database-deploy-scripts/2_Deploy_Databases.sh

```diff
@@ -45,6 +45,16 @@ then
     migration_publish_profile=".\ntbs-data-migration\Publish Profiles\Pre-production\DEV-migration.publish.xml"
     reporting_publish_profile=".\source\Publish Profiles\Pre-production\DEV-reporting.publish.xml"
     specimen_matching_publish_profile=".\source\Publish Profiles\Pre-production\DEV-specimen-matching.publish.xml"
+elif [[ $DEPLOY_ENVIRONMENT = "test" ]]
+then
+    migration_publish_profile=".\ntbs-data-migration\Publish Profiles\Pre-production\azure-migration.publish.xml"
+    reporting_publish_profile=".\source\Publish Profiles\Pre-production\azure-test-reporting.publish.xml"
+    specimen_matching_publish_profile=".\source\Publish Profiles\Pre-production\azure-test-specimen-matching.publish.xml"
+elif [[ $DEPLOY_ENVIRONMENT = "azure-uat" ]]
+then
+    migration_publish_profile=".\ntbs-data-migration\Publish Profiles\Pre-production\azure-migration.publish.xml"
```

Comment:
We have a separate `azure-uat-migration` database that we should use here (we introduced it a few weeks ago because we needed to split our test environments between branches)